.github/actions/rollback_release/index.js

```diff
@@ -0,0 +1,43 @@
+const { Octokit } = require("@octokit/core")
+const core = require('@actions/core');
+
+
+async function rollbackRelease() {
+  const octokit = new Octokit({
+      auth: process.env.GITHUB_TOKEN
+    })
+  
+  const { data } = await octokit.request('GET /repos/Bandwidth/api-docs/releases', {
+    owner: 'Bandwidth',
+    repo: 'api-docs'
+  })
+  
+  const cleanedData = []
```

Comment:
May be name it appropriately making it more expressive. For example, _failedAndSuccessfulReleases_ (if that's what we intend to hold in this list)

Reply:
Got it, may be `publishedReleases` in that case.